src/main/resources/io/vertx/ext/shell/shell.html

```diff
@@ -0,0 +1,155 @@
+<!doctype html>
+<title>Vert.x Shell</title>
+
+<h1>Vert.x Shell</h1>
+
+<style>
```

Comment:
As the javascript is in a separated file, what about moving the style in a css file ?